src/ape/utils/abi.py

```diff
@@ -86,6 +123,7 @@ def length(struct) -> int:
         # NOTE: Should never be "_{i}", but mypy complains and we need a unique value
         [m.name or f"_{i}" for i, m in enumerate(types)],
         namespace={"__getitem__": get_item, "__eq__": is_equal, "__len__": length},
+        bases=(Struct,),
```

Comment:
Why use empty class as base? Just for subclass checks?

Reply:
Ya, thought about putting a method but then we have to use `getattr`, and I thought the subclass check seemed nicer
but am open to alternatives.

Reply:
I like it, it works. Just add a comment about that use case